### PR TITLE
NEXT-35970 - Docs for shopping lists

### PR DESCRIPTION
**Pull request by nguyenky**

### CODE REVIEW

**Review by bojanrajh — CHANGES_REQUESTED** [review]
Should we remove numeric filename prefixes (`nav.position` should be enough for proper placement in the sidebar)?

File: products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Entities
+
+### Shopping Lists
+
+Shopping lists represent a list of products prepared for a customer or a sales channel. They show basic information about the product list, such as the name, customer, sales channel and so on.
+
+### Line item
+
+The  Shopping list line item represents a set of products. These products are line items of the shopping list.
```
Comment (leduc92) [inline products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md:18]: It seems like that there're 2 space between The  Shopping

File: products/extensions/b2b-components/shopping-lists/02-api-and-pricing.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+nav:
+  title: API & Pricing
+  position: 30
+
+---
+
+# Store API
```
Comment (leduc92) [inline products/extensions/b2b-components/shopping-lists/02-api-and-pricing.md:8]: Should we just create a link to our store API docs? 
https://shopware.stoplight.io/docs/store-api/c9849725606fd-create-new-shopping-list

File: products/extensions/b2b-components/shopping-lists/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+nav:
+  title: Shopping lists
+  position: 10
+
+---
+
+# Shopping lists
+
+The Shopping List application is designed to help users easily manage their shopping needs. It allows users to create, edit, and organize their shopping lists efficiently. With features such as item categorization, quantity tracking, and the ability to mark items as purchased, the Shopping List ensures that users can shop with ease and never forget essential items. Whether you are planning for a weekly grocery trip or a special event, the Shopping List app provides a simple and intuitive solution to keep track of all your shopping requirements.
```
Comment (leduc92) [inline products/extensions/b2b-components/shopping-lists/index.md:10]: List with `s`. 

IMO `Shopping Lists  component` is better than the `Shopping Lists  application`

File: products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md (hunk 1 above)
Comment (leduc92) [reply products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md:18]: Shopping Lists

File: products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+nav:
+  title: Entities & Schema
+  position: 20
+
+---
+
+# Entities and schema
```
Comment (sushmangupta) [inline products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md:8]: @leduc92 , @nguyenky - I would like to discuss about the document structuring w.r.t B2B components.

Comment (sushmangupta) [reply products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md]: After conversation with Duc and Nam, yesterday. I will rearrange the structure as per this : https://shopware.atlassian.net/browse/DX-592. After that will restructure this PR too.

**Comment by sushmangupta** [comment]
This PR info has been added in this PR: https://github.com/shopware/docs/pull/1407/files. Hence to avoid duplicity, I will close this.

**Pull request closed without merging**